src/dowel/csv_output.py

```diff
@@ -42,20 +42,42 @@ def record(self, data, prefix=''):
                 self._writer.writeheader()
 
             if to_csv.keys() != self._fieldnames:
-                self._warn('Inconsistent TabularInput keys detected. '
```

Comment:
Now that you've deleted the last call to `_warn`, is there any other code you can delete?